hikari/impl/rest.py

```diff
@@ -1054,6 +1055,7 @@ async def edit_channel(
         body.put("auto_archive_duration", auto_archive_duration, conversion=time.timespan_to_int)
         body.put("locked", locked)
         body.put("invitable", invitable)
+        body.put_array("applied_tags", applied_tags, conversion=self._entity_factory.serialize_forum_tag)
```

Comment:
This is missing a test case

Reply:
That should be covered by the `TestGuildThreadChannel.test_thread_edit` test. However, as I am removing the `GuildThreadChannel.edit` method, I will also remove that test. In its place, I will make the necessary changes to the `TestGuildChannel.test_edit` test.

Reply:
Realized there are tests for impl. Updating the tests.